xbmc/network/EventClient.cpp

```diff
@@ -370,7 +370,7 @@ bool CEventClient::OnPacketBUTTON(CEventPacket *packet)
   float famount = 0;
   bool active = (flags & PTB_DOWN) ? true : false;
   
-  CLog::Log(LOGDEBUG, "EventClient: button code %d %s", bcode, active ? "pressed" : "released");
+  CLog::Log(LOGDEBUG, "EventClient: button %s code %d %s", button, bcode, active ? "pressed" : "released");
```

Comment:
_untested:_
```suggestion
  CLog::Log(LOGDEBUG, "EventClient: button: \"{}\" code: \"{}\" action: \"{}\"", button, bcode, active ? "pressed" : "released");
```


i would suggest to change the logged line to make it more clear than:
`DEBUG: EventClient: button enter code 46255 pressed`

Reply:
button strings should also be surrounded by "" quotes to detect when something goes wrong in case an empty string shows up